src/game_api/memory.hpp

```diff
@@ -95,82 +220,28 @@ FunT* vtable_find(T* obj, size_t index)
         return static_cast<FunT*>(nullptr);
     return reinterpret_cast<FunT*>(&ptr[0][index]);
 }
-}; // namespace
 
-class ExecutableMemory

```

Comment:
Diff is very difficult to read here, I will get back to it when you moved the two classes back  :P